composer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"require-dev": {
```

Comment:
Please add "name": "litespeedtech/lscache_wp",

Reply:
@szepeviktor thank you! added